### PR TITLE
Waiting list

### PR DESCRIPTION
**Pull request by kevinprince**

### CODE REVIEW

File: source/blog/2013-06-11-its-a-sell-out.html.markdown [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Its a SELL OUT!
```
Comment (mseckington) [inline source/blog/2013-06-11-its-a-sell-out.html.markdown]: "It's"

File: source/blog/2013-06-11-its-a-sell-out.html.markdown [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+date: 2013-06-11 15:45 UTC
+tags:
+---
+
+In an amazing time of INSERT TIME minutes you all snapped up the final 200 hundred tickets! Now if you did not get a ticket theirs
```
Comment (mseckington) [inline source/blog/2013-06-11-its-a-sell-out.html.markdown]: final word should be "there's"

File: source/blog/2013-06-11-its-a-sell-out.html.markdown [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+date: 2013-06-11 15:45 UTC
+tags:
+---
+
+In an amazing time of INSERT TIME minutes you all snapped up the final 200 hundred tickets! Now if you did not get a ticket theirs 
+still a chance you can come to HACKED. We have now launched the waiting list which is available on the <a href="/waitinglist">waiting list</a> page.
+
+**How will it work?**
+
+From early July we will start asking people who can't make HACKED to return their tickets. From this pool we randomly allocate these tickets to those on the waiting list. If you are lucky enough to receive one of these tickets you will have a 24 hour window in which to complete your sign-up. If you miss the 24 hour windows the ticket will be reallocated to someone else on the list.
```
Comment (mseckington) [inline source/blog/2013-06-11-its-a-sell-out.html.markdown]: final "windows" needs to be "window"

File: source/questions.html.erb [hunk 4]
```diff
@@ -40,8 +40,8 @@
 
   <!-- Visas and other papers -->
   <h2>Can you provide visa letters?</h2>
-  <p>Sadly we are unable to provide any bespoke letters or paperwork visas. If your unsure if you need a visa to travel to the
-    United Kingdom you should speak to your like British Consulate or check <a href="http://www.fco.gov.uk">FCO.gov.uk</a>.</p>
+  <p>Sadly we are unable to provide any bespoke letters or paperwork for visas. If your unsure if you need a visa to travel to the
```
Comment (mseckington) [inline source/questions.html.erb]: Not really sure what changed here, but noticed that it needs to "If you're unsure".

Comment (kevinprince) [inline source/questions.html.erb]: Some words had got muddled so we fixing it.

Comment (mseckington) [inline source/questions.html.erb]: Ah kay.

Still needs the "if you're unsure" fix though.

**Pull request merged**